django_celery_beat/locale/fr/LC_MESSAGES/django.po

```diff
@@ -7,7 +7,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2022-12-22 19:03+0000\n"
+"POT-Creation-Date: 2024-06-25 11:24+0200\n"
 "PO-Revision-Date: 2022-10-14 23:48+0200\n"
```

Comment:
Should we be modifying the revision date instead of the creation date?

Reply:
I just left what `django-admin makemessages -l fr` outputed but it wasn't a big change so I just rolled back that